upgrade/scripts/sls/sls_utils/Networks.py

```diff
@@ -250,19 +251,6 @@ def __init__(self, default_route_network_name="CMN"):
         self.__system_default_route = default_route_network_name
         self.mtu(network_mtu=9000)
 
-    def system_default_route(self, default_route_network_name=None):
```

Comment:
I don't think you should be changing anything in sls_utils unless something is totally broke.  It supports all the structures consistently.  Looks like you removed a function in the core lib to maintain the BICAN structure.